webpack.common.config.ts

```diff
@@ -531,6 +534,17 @@ module.exports = {
       minify: htmlMinifyConfig,
       inject: false
     }),
+    new HtmlWebpackPlugin({
+      chunks: ['learner_group_viewer'],
+      filename: 'view-learner-group-page.mainpage.html',
+      hybrid: true,
+      meta: defaultMeta,
+      template:
+        commonPrefix + '/pages/learner-group-pages/view-group/' +
+        'view-learner-group-page.mainpage.html',
+      minify: htmlMinifyConfig,
+      inject: false
+    }),
```

Comment:
@paprajapati9 Can these pages be added using angular router and are not downgraded? //cc @ashutoshc8101 @srijanreddy98

Reply:
@Rijuta-s  Can you please elaborate a bit, I am not familiar with adding pages that way.

Reply:
@Rijuta-s Have moved adding of page from webpack to app route module. Thanks!